### PR TITLE
[TF] Update e2e tests

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

Update e2e tests

### Related tickets

128812

### CODE REVIEW

**Review by alexsu52 — COMMENTED** [review]
Could you please provide the e2e green build number?

**Comment by AlexanderDokuchaev** [comment]
/job/ci-maintenance/job/rmkh-dev/job/NNCF/job/nightly/job/tf_full/60

File: tests/tensorflow/data/ac_configs/mobilenet_v2_imagenet.yml [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+        adapter: classification
+    datasets:
+      - name: imagenet_1000_classes
+        preprocessing:
+          - type: crop_image_with_padding
+            size: 224
+            crop_padding: 32
+          - type: rgb_to_bgr
+          - type: normalization
+            mean: '127.5, 127.5, 127.5'
```
Comment (alexsu52) [inline tests/tensorflow/data/ac_configs/mobilenet_v2_imagenet.yml:15]:
```suggestion
            mean: 127.5, 127.5, 127.5
```

**Review by alexsu52 — APPROVED** [review]
LGTM

**Pull request merged**